src/test/resources/__files/mocks/errorResponse.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
Are these json files left over from some previous test? Don't we need them anymore?

Reply:
Yes, the tests using those files are only part of internal SDK, so getting rid of them in public sdk.